src/main/java/creativei/service/impl/InquiryServiceImpl.java

```diff
@@ -111,4 +120,66 @@ public Inquiry updateAll(List<Inquiry> inquiries) {
         Long cityId = filterVo.getCityVo().getId();
         return inquiryCustomDao.findByFilters(status, caseIndex, cityId);
     }
+
+    @Override
+    public ResponseObject getInquiryCount() throws ParseException {
+        InquiryCountVo inquiryCountVo=null;
+        try {
+            inquiryCountVo= new InquiryCountVo();
+            inquiryCountVo.setDailyCount(getDailyCount());
+            inquiryCountVo.setWeekCount(getWeeklyCount());
+            inquiryCountVo.setMonthCount(getMonthlyCount());
+            inquiryCountVo.setHotLeadCount(getHotLeadCount());
+            inquiryCountVo.setEnrolledCount(getEnrolledCount());
+            return ResponseObject.getResponse(inquiryCountVo);
+        }catch (Exception e){
+            logger.error("Cause: "+e.getCause().getCause().getMessage());
+            return ResponseObject.getResponse(ExceptionType.RESULTSET_GENERATION_EXCEPTION.getMessage(),ExceptionType.RESULTSET_GENERATION_EXCEPTION.getCode());
+        }
+    }
+
+
+    private Integer getDailyCount() throws ParseException {
+        logger.info("getDailyCount method");
```

Comment:
We are converting the date to string and the converting the same string to date. No need of string.

Reply:
new Date gives the date with current time i have to make sure that date should start with time 00:00 this is the reason why i use string.

Reply:
Use calendar class setters. Using that you can set time to 0

Reply:
issue resolved in commit 2df898a